btree/btree.c

```diff
@@ -1786,16 +1833,24 @@ static int64_t node_alloc(struct node_op *op, struct td *tree, int size,
 	return nxt_state;
 }
 
-#ifndef __KERNEL__
 static int64_t node_free(struct node_op *op, struct nd *node,
 			 struct m0_be_tx *tx, int nxt)
 {
 	int shift = node->n_type->nt_shift(node);
+
+	node_refcnt_update(node, false);
```

Comment:
I assume the code which will wait for all the references to have gone to 0 will be coming in a later PR.

Reply:
Yes. Will  merge those changes once they are pushed.